packages/terser/src/index.js

```diff
@@ -43,20 +44,52 @@ function terserDirectory(input) {
     fs.mkdirSync(output);
   }
 
+  let work = [];
+  let concurrency = (os.cpus().length - 1 || 1);
+  let active = 0;
+  let errors = [];
+
+  function exec([inputFile, outputFile]) {
+    active++;
+    let args =
+        [require.resolve('terser/bin/uglifyjs'), inputFile, '--output', outputFile, ...residual];
+
+    spawn(process.execPath, args).then((data) => {
+      if (data.code) {
+        errors.push({file: inputFile, out: data.out + '', err: data.error + '', code: data.code})
+      }
+
+      console.log('finished: ', inputFile, '\n', data.out + '');
```

Comment:
we should be silent on success - maybe use log_verbose if you want to leave this in so that users bug reports include this

Reply:
but how will they depend on my observable outputs! (jk)
sounds good.